### PR TITLE
feat: UI Tweaks to Card Design in Grant Detail

### LINKED ISSUE #3373: [Issue]: UI tweaks to card design related to issue 3154
### Why is this issue important?

Updates following the first release of card design UI seen in staging. 

### Current State

**Card Design**
Border Radius: 4px (.25rem) -> Change to 8px (.5rem)

**Main Card (Grant Detail)**
Margins for primary (main content) card: 20px -> Change to 24px (1.5rem)
image
Back Button Row: Change top margin to 0px (24px total with card margin); bottom margin to 24px.
Grant Title Margins: Top: 16px (1rem), bottom: 24px (1.5rem).
Spacing between button row (view on grants, copy, print): 24px (1.5rem).
Make the button left-aligned to the left margin of the card (24px from edge)

**Secondary Card (Share, Status, Grant Activity, etc)**
Top/Bottom Margin: Change to 24px (1.5rem) -- Keeps the cards visually aligned in both columns (main and side).
Left/Right Margin: Change to 16px (1rem)-- to meet the narrower column sizing, shortening the side margins gives us more content space to work with.

image

Margin between search bar and entry for share / status card: 24px (1.5rem)
image
Total spacing between entries: 24px (1.5rem)
- May need to remove padding on email address line (currently at 16px)
image

### Expected State

When user visits X page...

### Implementation Plan

The following functions need to be added...

### Relevant Code Snippets

_No response_

### PR DESCRIPTION
**Pull request by sushilrajeeva**
### Ticket #3373

Fixes #3373 

## Description
This PR is part of #3373 which implements UI tweaks to the Grant Detail page to align with updated design specifications. Changes include adjusting margins, spacing, and alignment across card components to improve visual consistency and responsiveness.

This is related to issues #3154 and #3240 

## Screenshots / Demo Video
Screenshot 2024-08-23 at 3 55 06 PM

### Automated and Unit Tests
- [ ] Added Unit tests

### Manual tests for Reviewer
- [ ] Added steps to test feature/functionality manually

## Checklist
- [x] Provided ticket and description
- [x] Provided screenshots/demo
- [ ] Provided testing information
- [ ] Provided adequate test coverage for all new code
- [x] Added PR reviewers

### CODE REVIEW

File: packages/client/src/views/GrantDetailsView.vue [hunk 1]
```diff
@@ -611,6 +614,7 @@ export default {
 
   .grant-details-content {
     grid-area: content;
+    /* margin: 24px; */
```
Comment (joshgarza) [inline packages/client/src/views/GrantDetailsView.vue:617]: _nit: can we remove the commented out margin?_

**Review by joshgarza — APPROVED** [review]
Just left one nitpick that isn't relevant to the function of your ticket. Looks good to me!

File: packages/client/src/views/GrantDetailsView.vue (hunk 1 above)
Comment (sushilrajeeva) [reply packages/client/src/views/GrantDetailsView.vue:617]: Sure.

File: packages/client/src/views/GrantDetailsView.vue [hunk 2]
```diff
@@ -590,8 +593,8 @@ export default {
   row-gap: 48px;
   .grant-details-back-link {
     grid-area: back-link;
-    margin-top: 28px;
-    margin-bottom: 5px;
+    margin-top: 0px;
```
[the remark this answers is not in the document]
Comment (sushilrajeeva) [reply packages/client/src/views/GrantDetailsView.vue]: Hi Greg, I have replaced px with rem conversion as suggested.

**Review by greg-adams — APPROVED** [review]

**Pull request merged**